spec/models/thermometer_widget_spec.rb

```diff
@@ -1,41 +1,25 @@
 describe ThermometerWidget do
 
-  let(:content) { {
+  let(:params) { {
     goal: 10000,
     count: 500,
-    autoincrement: true
+    autoincrement: true,
+    page_display_order: 1
   } }
-  let(:string_content) { {
-    goal: "12345",
-    count: "45",
-    autoincrement: "0"
-  } }
```

Comment:
Why did you axe this test? If you're refactoring the type casting, I would expect to see at least the existing test passing, if not some more comprehensive type tests (I didnt have ones for array or hash, for example)